src/ui/pages/equipment/asset/Asset.js

```diff
@@ -364,6 +365,29 @@ const Asset = () => {
                 ignore: !isCernMode || !getTabAvailability(tabs, TAB_CODES.EQUIPMENT_GRAPH_ASSETS),
                 initialVisibility: getTabInitialVisibility(tabs, TAB_CODES.EQUIPMENT_GRAPH_ASSETS)
             },
+            {
+                id: 'PARTSASSOCIATED',
+                label: 'Parts Associated',
+                isVisibleWhenNewEntity: false,
+                maximizable: true,
+                render: ({ isMaximized }) =>
+                    <EAMGridTab
+                        gridName={'BSPARA'}
+                        objectCode={equipment.code + '#' + equipment.organization}
+                        paramNames={['param.valuecode']}
+                        additionalParams={{'param.entity': 'OBJ'}}
+                        showGrid={isMaximized}
+                        rowCount={100}
+                        gridContainerStyle={isMaximized ? { height: `${document.getElementById('entityContent').offsetHeight - 220}px`} : {}}
+                    >   
+                    </EAMGridTab>
+                ,
+                column: 2,
+                order: 35,
+                summaryIcon: PartPlusIcon,
+                ignore: !getTabAvailability(tabs, TAB_CODES.PARTS_ASSOCIATED),
+                initialVisibility: getTabInitialVisibility(tabs, TAB_CODES.PARTS_ASSOCIATED)
+            },
```

Comment:
This object is really similar throughout. Can we externalize it?

Reply:
@ricardo-j-almeida done